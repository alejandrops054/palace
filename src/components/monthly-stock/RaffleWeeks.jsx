/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos Dev01
Comentarios: xxxxx
*/

import React from "react";
import {
  Container,
  Col,
  Row,
  Button,
  InputGroup,
  InputGroupAddon,
  InputGroupText,
  Input,
  Label,
} from "reactstrap";

function RaffleWeeks() {
  return (
    <Container fluid>
      <Row>
        <Col>
          <h1>Raffle Weeks</h1>
        </Col>
        <Col>
          <Row className="mt-2">
            <Col lg={{ size: 4, offset: 6 }} md={{ size: 3, offset: 6 }}>
              <Row>
                <Button block>Get Raffle Week</Button>
              </Row>
            </Col>
          </Row>
          <Row className="mt-1">
            <Col lg={{ size: 4, offset: 6 }} md={{ size: 3, offset: 6 }}>
              <Row>
                <Button block>New</Button>
              </Row>
            </Col>
          </Row>
        </Col>
      </Row>
      <Row>
        <Row>
          <Col lg={{ size: 2 }} md={{ size: 2 }}>
            <Label>
              Project: <Input type="text" />
            </Label>
          </Col>
          <Col className="mt-4" lg={{ size: 5 }}>
            <Input type="select" />
          </Col>
        </Row>
        <Row>
          <Col lg={{ size: 2 }} md={{ size: 2 }}>
            <Label>
              Office: <Input type="text" />
            </Label>
          </Col>
          <Col className="mt-4" lg={{ size: 5 }}>
            <Input type="select" />
          </Col>
        </Row>
        <Row>
          <Col lg={{ size: 2 }} md={{ size: 2 }}>
            <Label>
              Folio: <Input type="text" />
            </Label>
          </Col>
        </Row>
        <Row>
          <Col lg={{ size: 3 }} md={{ size: 3 }}>
            <Label>
              Application: <Input type="text" />
            </Label>
          </Col>
        </Row>
        <Row>
          <Col lg={{ size: 5 }} md={{ size: 5 }}>
            <Row>
              <Label>
                First: <Input type="text" />
              </Label>
            </Row>
          </Col>
        </Row>
        <Row>
          <Col lg={{ size: 5 }} md={{ size: 5 }}>
            <Row>
              <Label>
                Last: <Input type="text" />
              </Label>
            </Row>
          </Col>
        </Row>
        <Row>
          <Col lg={{ size: 10 }} md={{ size: 10 }}>
            <Row>
              <Label>
                Address: <Input type="text" />
              </Label>
            </Row>
          </Col>
        </Row>
        <Row>
          <Col lg={{ size: 10 }} md={{ size: 8 }}>
            <Row>
              <InputGroup>
                <Label>
                  {" "}
                  Phone: <Input type="number" />{" "}
                </Label>
                <InputGroupAddon addonType="prepend">
                  <InputGroupText className="mt-4 ml-2">.</InputGroupText>
                </InputGroupAddon>
              </InputGroup>
            </Row>
          </Col>
        </Row>
        <Row>
          <Col lg={{ size: 10 }} md={{ size: 8 }}>
            <Row>
              <InputGroup>
                <Label>
                  {" "}
                  City: <Input type="text" />{" "}
                </Label>
                <InputGroupAddon addonType="prepend">
                  <InputGroupText className="mt-4 ml-2">..</InputGroupText>
                </InputGroupAddon>
              </InputGroup>
            </Row>
          </Col>
        </Row>
        <Row>
          <Col lg={{ size: 10 }} md={{ size: 10 }}>
            <Row>
              <Label>
                State: <Input type="text" />
              </Label>
            </Row>
          </Col>
        </Row>
        <Row>
          <Col lg={{ size: 5 }} md={{ size: 5 }}>
            <Row>
              <Label>
                Country: <Input type="text" />
              </Label>
            </Row>
          </Col>
          <Col lg={{ size: 3 }} md={{ size: 3 }}>
            <Row>
              <Label>
                Zip: <Input type="text" />
              </Label>
            </Row>
          </Col>
        </Row>
        <Row>
          <Col lg={{ size: 3 }} md={{ size: 3 }}>
            <Row>
              <Label>
                Sale Date: <Input type="date" />
              </Label>
            </Row>
          </Col>
          <Col lg={{ size: 3, offset: 1 }} md={{ size: 3, offset: 1 }}>
            <Row>
              <Label>
                Venc Yrs: <Input type="number" />
              </Label>
            </Row>
          </Col>
          <Col lg={{ size: 3, offset: 1 }} md={{ size: 3, offset: 1 }}>
            <Row>
              <Label>
                Validate: <Input type="date" />
              </Label>
            </Row>
          </Col>
        </Row>
        <Row>
          <Col lg={{ size: 6 }} md={{ size: 6 }}>
            <Row>
              <Label>
                Language: <Input type="select" />
              </Label>
            </Row>
          </Col>
        </Row>
      </Row>
    </Container>
  );
}

export default RaffleWeeks;
