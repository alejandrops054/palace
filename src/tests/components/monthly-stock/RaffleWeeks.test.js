import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import RaffleWeeks from '../../../components/monthly-stock/RaffleWeeks'

describe('Raffle Weeks component',() => {
    test('debe mostrar snapshot', () => {
        
        const wrapper = shallow(<RaffleWeeks/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})