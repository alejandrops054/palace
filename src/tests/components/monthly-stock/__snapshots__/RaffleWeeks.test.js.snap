// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Raffle Weeks component debe mostrar snapshot 1`] = `
<Container
  fluid={true}
  tag="div"
>
  <Row
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <h1>
        Raffle Weeks
      </h1>
    </Col>
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Row
        className="mt-2"
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          lg={
            Object {
              "offset": 6,
              "size": 4,
            }
          }
          md={
            Object {
              "offset": 6,
              "size": 3,
            }
          }
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Row
            tag="div"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            <Button
              block={true}
              color="secondary"
              tag="button"
            >
              Get Raffle Week
            </Button>
          </Row>
        </Col>
      </Row>
      <Row
        className="mt-1"
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          lg={
            Object {
              "offset": 6,
              "size": 4,
            }
          }
          md={
            Object {
              "offset": 6,
              "size": 3,
            }
          }
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Row
            tag="div"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            <Button
              block={true}
              color="secondary"
              tag="button"
            >
              New
            </Button>
          </Row>
        </Col>
      </Row>
    </Col>
  </Row>
  <Row
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 2,
          }
        }
        md={
          Object {
            "size": 2,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Label
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          Project: 
          <Input
            type="text"
          />
        </Label>
      </Col>
      <Col
        className="mt-4"
        lg={
          Object {
            "size": 5,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Input
          type="select"
        />
      </Col>
    </Row>
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 2,
          }
        }
        md={
          Object {
            "size": 2,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Label
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          Office: 
          <Input
            type="text"
          />
        </Label>
      </Col>
      <Col
        className="mt-4"
        lg={
          Object {
            "size": 5,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Input
          type="select"
        />
      </Col>
    </Row>
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 2,
          }
        }
        md={
          Object {
            "size": 2,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Label
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          Folio: 
          <Input
            type="text"
          />
        </Label>
      </Col>
    </Row>
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 3,
          }
        }
        md={
          Object {
            "size": 3,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Label
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          Application: 
          <Input
            type="text"
          />
        </Label>
      </Col>
    </Row>
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 5,
          }
        }
        md={
          Object {
            "size": 5,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Row
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            First: 
            <Input
              type="text"
            />
          </Label>
        </Row>
      </Col>
    </Row>
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 5,
          }
        }
        md={
          Object {
            "size": 5,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Row
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Last: 
            <Input
              type="text"
            />
          </Label>
        </Row>
      </Col>
    </Row>
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 10,
          }
        }
        md={
          Object {
            "size": 10,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Row
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Address: 
            <Input
              type="text"
            />
          </Label>
        </Row>
      </Col>
    </Row>
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 10,
          }
        }
        md={
          Object {
            "size": 8,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Row
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <InputGroup
            tag="div"
          >
            <Label
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
               
              Phone: 
              <Input
                type="number"
              />
               
            </Label>
            <InputGroupAddon
              addonType="prepend"
              tag="div"
            >
              <InputGroupText
                className="mt-4 ml-2"
                tag="span"
              >
                .
              </InputGroupText>
            </InputGroupAddon>
          </InputGroup>
        </Row>
      </Col>
    </Row>
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 10,
          }
        }
        md={
          Object {
            "size": 8,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Row
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <InputGroup
            tag="div"
          >
            <Label
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
               
              City: 
              <Input
                type="text"
              />
               
            </Label>
            <InputGroupAddon
              addonType="prepend"
              tag="div"
            >
              <InputGroupText
                className="mt-4 ml-2"
                tag="span"
              >
                ..
              </InputGroupText>
            </InputGroupAddon>
          </InputGroup>
        </Row>
      </Col>
    </Row>
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 10,
          }
        }
        md={
          Object {
            "size": 10,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Row
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            State: 
            <Input
              type="text"
            />
          </Label>
        </Row>
      </Col>
    </Row>
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 5,
          }
        }
        md={
          Object {
            "size": 5,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Row
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Country: 
            <Input
              type="text"
            />
          </Label>
        </Row>
      </Col>
      <Col
        lg={
          Object {
            "size": 3,
          }
        }
        md={
          Object {
            "size": 3,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Row
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Zip: 
            <Input
              type="text"
            />
          </Label>
        </Row>
      </Col>
    </Row>
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 3,
          }
        }
        md={
          Object {
            "size": 3,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Row
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Sale Date: 
            <Input
              type="date"
            />
          </Label>
        </Row>
      </Col>
      <Col
        lg={
          Object {
            "offset": 1,
            "size": 3,
          }
        }
        md={
          Object {
            "offset": 1,
            "size": 3,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Row
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Venc Yrs: 
            <Input
              type="number"
            />
          </Label>
        </Row>
      </Col>
      <Col
        lg={
          Object {
            "offset": 1,
            "size": 3,
          }
        }
        md={
          Object {
            "offset": 1,
            "size": 3,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Row
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Validate: 
            <Input
              type="date"
            />
          </Label>
        </Row>
      </Col>
    </Row>
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        lg={
          Object {
            "size": 6,
          }
        }
        md={
          Object {
            "size": 6,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Row
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Language: 
            <Input
              type="select"
            />
          </Label>
        </Row>
      </Col>
    </Row>
  </Row>
</Container>
`;
